webapp/src/features/Settings/Settings.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Row, Col } from 'react-bootstrap'
+import YourInformation from './YourInformation'
+import UpdatePassword from './UpdatePassword'
+export default function Settings() {
+	return (
+		<div>
+			<Row>
+				<Col lg={4} xl={5}>
+					<YourInformation />
+				</Col>
+				<Col lg={4} xl={5}>
+					<UpdatePassword />
+				</Col>
+			</Row>
+
+			<footer className=' py-4'>
```

Comment:
This should be as seperate component, and used in App.js, so that it will be same for all screens